### ISSUE TITLE
Spell-check does not check to end of long files

### ISSUE THREAD

**Issue by gregvanl** [Microsoft/vscode-spell-check#15]
For example, open https://github.com/Microsoft/vscode-docs/blob/master/release-notes/latest.md for 0.10.7 release and you'll see the correction don't go to the end of the document.

**Comment by seanmcbreen** [comment]
I'm alive :)  

I quickly looked into this and the issue is there are to many errors :) We cap diagnostics at 250 results and this baby has a few more.  
![too many](https://cloud.githubusercontent.com/assets/1051190/12805214/78e5ce36-caae-11e5-90da-707d21557265.PNG)

I think the best fix would be to re-introduce a setting for ignore reg-exp sections e.g. do not report errors in links as that would cut down a lot of the errors.  This would also resolve issue #11 and was in the original version.  

I can look into this - will stop me getting to rusty.

Say hi to all!

**Comment by seanmcbreen** [comment]
I added in support for this in 0.5.0.

I suggest you add the following RegExp's to the settings file.  then you should be all good to go...

```
    "ignoreRegExp": [
        "/`(kb.*?)`/g",
        "/(http\\\\S*)/gm",
        "/\\\\(.*.png\\\\)/g",
        "/\\\\(.*.gif\\\\)/g",
        "/\\\\(.*.md\\\\)/g",
        "/\\\\(.*.jpg\\\\)/g",
        "/^((`{3}\\\\s*)(\\\\w+)?(\\\\s*([\\\\w\\\\W]+?)\\\\n*)\\\\2)\\\\n*(?:[^\\\\S\\\\w\\\\s]|$)/gm"
    ]
```

LMK if there are any issues - closing this for now :)
